habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -237,6 +237,7 @@ class PolicyConfig(HabitatBaselinesBaseConfig):
     hierarchical_policy: HierarchicalPolicy = MISSING
     ovrl: bool = False
     no_downscaling: bool = False
+    use_augmentations: bool = False
```

Comment:
Should we enable them by default in our configs?

Reply:
Lets do it after we have tested it once on the new dataset